index.bs

```diff
@@ -4422,8 +4452,9 @@ closed</dfn> steps given |window|, |accepted| and optional |user text|
 
         browsingContext.UserPromptOpenedParameters = {
           context: browsingContext.BrowsingContext,
-          type: "alert" / "confirm" / "prompt" / "beforeunload",
+          handler: "accept" / "dismiss" / "ignore",
           message: text,
+          type: "alert" / "confirm" / "prompt" / "beforeunload",
```

Comment:
Was there actually a reason why we didnt' add the `type` to the `browsingContext.UserPromptClosedParameters` as well? I think it would be good to know for the client which type of alert was closed.

Reply:
We could. In theory you can know as long as you subscribe to both events, since the prompt that's closed is always the last prompt opened for that context (as long as we don't support multiple prompts). I don't really know why you'd subscribe to the close event but not the open event, but we can make things a bit easier for anyone who does that.

Reply:
Sure, but one option I could imagine is that you have set the handler to `accept` or `dismiss` to automatically close those prompts, but just want to know which kind of prompts were open at which times. But that might not be used that often.